diff.py

```diff
@@ -1330,6 +1330,114 @@ def dump_binary(
     )
 
 
+class RelocationProcessor:
+    def __init__(self, config: Config) -> None:
```

Comment:
better pass arch as an argument here than in process

Reply:
I removed the argument from process() altogether, as it can be derived from `config`.